modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IOParMap2.kt

```diff
@@ -12,7 +12,7 @@ import kotlin.coroutines.CoroutineContext
 /** Mix-in to enable `parMapN` 2-arity on IO's companion directly. */
 interface IOParMap2 {
 
-  fun <A, B, C> parMapN(ctx: CoroutineContext, fa: IOOf<A>, fb: IOOf<B>, f: (A, B) -> C): IO<C> = IO.Async { conn, cb ->
+  fun <A, B, C> parMapN(ctx: CoroutineContext, fa: IOOf<A>, fb: IOOf<B>, f: (A, B) -> C): IO<C> = BIO.Async { conn, cb ->
```

Comment:
This is why I'd rather we kept the name as `IO`. We have the examples and people don't mind, it'll make the migration easier, and we don't have to rename to RIO or something once we add reader.

Reply:
Oh actually you're happy you say that! I would also be in favor of keeping `IO` to be honest, I thought we wanted to keep the current API alive as much as possible.

It seems that'll far apart since a lot of functions need to become extensions since Kotlin doesn't have variance for functions.

Reply:
I.e. `flatMap` became an extension function so that now requires an import.
That will be true for all combinators that have E` in the `in`position.

Reply:
If possible, I'd also rather keeping the `IO` name for all (IO, BIO and RIO).

Reply:
Agreed on keeping IO as single name. f we do that we need to write a small migration guide to broken apis that would now require E specified

Reply:
`IO<A>` is aliased to `BIO<Nothing, A>` so they shouldn't need to, right?

Reply:
@pakoito after making the change. Not sure if that'll make it easier to migrate.

Reply:
@JorgeCastilloPrz I think Paco wants `IO<Nothing, A>` and `IO<E, A>`.